crates/sui-core/src/authority/authority_store.rs

```diff
@@ -787,6 +787,7 @@ impl AuthorityStore {
             written,
             deleted,
             events,
+            ..
```

Comment:
Better to add the unused fields explicitly, I think.

Reply:
I left this field private so could not do it but I guess it does not matter much if we do make it public to address this.